tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1414,6 +1414,38 @@ def func(self, xp, a):
             return a.sum(axis=self.axis, keepdims=self.keepdims)
 
 
+@op_utils.op_test(['native:0'])
+class TestSumStability(op_utils.NumpyOpTest):
+
+    skip_backward_test = True
+    skip_double_backward_test = True
+
+    def generate_inputs(self):
+        return numpy.full(2 ** 20, 0.1, dtype=numpy.float32),
```

Comment:
How about parameterizing the length of the array? We might be better to add the test for odd length.